guides/v15-mdc-migration.md

```diff
@@ -591,15 +591,14 @@ TODO(mmalerba): link to density docs once they exist.
 
 ### Tooltip
 
-* Accessibility: the background color is now completely opaque, preventing text bleed-through.
+* Background color is opaque instead of slightly transparent. This improves accessibility.
 
-* The default font-size has changed from `10px` to `12px`.
+* Default font-size is `10px` instead of `12px`.
```

Comment:
I'm not sure if this is correct. I'm pretty sure the MDC version has a larger font size.

Reply:
Huh you're right. I wonder why we had that in there. Maybe that's Google Material?